Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/TopLevel/TopographyTopLevelConverterToSpeckle.cs

```diff
@@ -31,7 +31,8 @@ public override SOBR.RevitTopography Convert(DBA.TopographySurface target)
     {
       units = _contextStack.Current.SpeckleUnits,
       displayValue = _displayValueExtractor.GetDisplayValue(target),
-      elementId = target.Id.ToString().NotNull()
+      elementId = target.Id.ToString().NotNull(),
+      baseGeometry = null! //TODO: this can't be correct
```

Comment:
This is a good observation 👀 I have no answer for this yet

Reply:
ty